components/dictionary.txt

```diff
@@ -46,6 +46,9 @@ CCDL's
 CDKN
 cDNA
 CellMarker
+celltype
+celltypes
+Celltype
```

Comment:
Let's use two words for "cell type" unless talking about code, where backticks should prevent spelling errors
```suggestion
```